ext/native/thin3d/VulkanRenderManager.cpp

```diff
@@ -401,8 +401,43 @@ void VulkanRenderManager::CopyFramebufferToMemorySync(VKRFramebuffer *src, int a
 
 	FlushSync();
 
-	// Need to call this after FlushSyfnc so the pixels are guaranteed to be ready in CPU-accessible VRAM.
-	queueRunner_.CopyReadbackBuffer(w, h, destFormat, pixelStride, pixels);
+	// Need to call this after FlushSync so the pixels are guaranteed to be ready in CPU-accessible VRAM.
```

Comment:
Would prefer this comment to stay together with the call to CopyReadbackBuffer.

Reply:
Oops, indeed.

-[Unknown]